pkg/odo/cli/pipelines/bootstrap.go

```diff
@@ -66,7 +64,10 @@ func (bo *BootstrapOptions) Validate() error {
 
 // Run runs the project bootstrap command.
 func (bo *BootstrapOptions) Run() error {
-	return pipelines.Bootstrap(bo.quayUsername, bo.baseRepo, bo.prefix)
+	options := pipelines.BootstrapOptions{
+		Prefix: bo.prefix,
+	}
+	return pipelines.Bootstrap(bo.quayUsername, bo.baseRepo, bo.githubToken, bo.quayIOAuthFilename, &options)
 }
 
```

Comment:
I would put all options (quayUserName, baseRepo, githubToken, quayIOAuthFilename) in the pipelines.BootstrapOptions and just pass that struct to pipelines.Bootstrap().

I would rename baseRepo to gitRepo.

Reply:
BootstrapOptions is a struct that has optional parameters that needs to be passed. I will rename baseRepo to gitRepo.

Reply:
Hm ...  Then, we should all the "required" options in a difference struct and pass it instead of 4 strings.

Reply:
I will rename it to BootstrapParameters

Reply:
You see the argument about only put "optional" options in BootstrapOptions does not hold any water.    See what are in the BootstrapOptions in this very same file.